Amazon A2I Fraud Detector.ipynb

```diff
@@ -0,0 +1,796 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Amazon Augmented AI (Amazon A2I) integration with Amazon Fraud Detector"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Visit https://github.com/aws-samples/amazon-a2i-sample-jupyter-notebooks for all A2I Sample Notebooks\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. [Introduction](#Introduction)\n",
+    "2. [Prerequisites](#Setup)\n",
+    "    1. [Workteam](#Workteam)\n",
+    "    2. [Notebook Permission](#Notebook-Permission)\n",
+    "3. [Client Setup](#Client-Setup)\n",
+    "4. [Create Control Plane Resources](#Create-Control-Plane-Resources)\n",
+    "    1. [Create Human Task UI](#Create-Human-Task-UI)\n",
+    "    2. [Create Flow Definition](#Create-Flow-Definition)\n",
+    "5. Scenario: When Activation Conditions are met, and a Human Loop is created\n",
+    "    1. [Check Status of Human Loop](#Check-Status-of-Human-Loop)\n",
+    "    2. [Wait For Workers to Complete Task](#Wait-For-Workers-to-Complete-Task)\n",
+    "    3. [Check Status of Human Loop](#Check-Status-of-Human-Loop)\n",
+    "    4. [View Task Results](#View-Task-Results)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "Amazon Augmented AI (Amazon A2I) makes it easy to build the workflows required for human review of ML predictions. Amazon A2I brings human review to all developers, removing the undifferentiated heavy lifting associated with building human review systems or managing large numbers of human reviewers. \n",
+    "\n",
+    "Amazon A2I provides built-in human review workflows for common machine learning use cases, such as content moderation and text extraction from documents, which allows predictions from Amazon Rekognition and Amazon Textract to be reviewed easily. You can also create your own workflows for ML models built on Amazon SageMaker or any other tools. Using Amazon A2I, you can allow human reviewers to step in when a model is unable to make a high confidence prediction or to audit its predictions on an on-going basis. Learn more here: https://aws.amazon.com/augmented-ai/\n",
+    "\n",
+    "In this tutorial, we will show how you can use Amazon A2I directly within your API calls to Amazon Rekognition's Detect Moderation Labels API. \n",
```

Comment:
This section needs to be changed to reflect fraud detector use case 

"In this tutorial, we will show how you can use Amazon A2I directly within your API calls to Amazon Rekognition's Detect Moderation Labels API."

Reply:
resolved in new pull request